addons/web/static/src/views/fields/properties/property_definition.xml

```diff
@@ -54,10 +54,10 @@
                             title="state.typeLabel" togglerClass="'btn btn-link d-flex p-0 w-100'">
                             <t t-set-slot="toggler">
                                 <div class="o_input_dropdown w-100 o_field_property_dropdown">
+                                    <img t-attf-src="/web/static/src/views/fields/properties/icons/{{state.propertyDefinition.type}}.png"/>
```

Comment:
Is it necessary to do this change because of dark mode? Does not seem related.

Not great to change the field templates in stable, even if the change has a low impact.

Reply:
yes it's needed, otherwise we can't apply a filter on it (and it's better to fix it that way, rather than duplicating all the images imo)

Reply:
Ok then, can you just explain in the commit message that we change from background-color to a real image tag so that we can apply the "filter" css function on it in dark mode?

Reply:
@awa-odoo Done :)